app/controllers/publishers_controller.rb

```diff
@@ -72,12 +73,13 @@ def create
       @publisher = verified_publisher
       PublisherLoginLinkEmailer.new(email: email).perform
       flash.now[:alert] = t(".email_already_active", email: email)
-      render :create_auth_token
+      params[:publisher_id] = @publisher.id
+      render :emailed_auth_token
     elsif @publisher.save
       PublisherMailer.verify_email(@publisher).deliver_later
       PublisherMailer.verify_email_internal(@publisher).deliver_later if PublisherMailer.should_send_internal_emails?
-      session[:created_publisher_id] = @publisher.id
-      redirect_to create_done_publishers_path
+      params[:publisher_id] = @publisher.id
```

Comment:
Now we rely on the publisher id being in the params to identify the publisher in PublishersController#resend_auth_email

Reply:
@nvonpentz It seems that the `emailed_auth_token` view relies on `@publisher.id` and not `params[:publisher_id]` to form the resend auth email path, so I don't think you need to set `params[:publisher_id]` here or on line 76 above.

Reply:
@dgeb You are correct, making changes